.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ on:
 env:
   MESON_VERSION: '0.63.3'
   EM_VERSION: '3.1.68'
-  PYTHON_VERSION: '3.11.8'
+  PYTHON_VERSION: '3.11'
```

Comment:
What's up with this change?

Reply:
The images we use currently provide the latest `3.11.x` dot-releases, which is something like `3.11.10`.
 
Basically, by specifying `3.11.8`, what would happen is that later revisions of `3.11.x` would be uninstalled, at this older dot-release was being installed. It would uninstall `3.11.10` and install `3.11.8`, which seemed really silly.

tl;dr: This speeds up the build process by about 10-15 seconds.